ring_doorbell/other.py

```diff
@@ -0,0 +1,287 @@
+# coding: utf-8
+# vim:sw=4:ts=4:et:
+"""Python Ring Other (Intercom) wrapper."""
+import json
+import logging
+import uuid
+
+from ring_doorbell.const import (
+    DOORBELLS_ENDPOINT,
+    HEALTH_DOORBELL_ENDPOINT,
+    INTERCOM_ALLOWED_USERS,
+    INTERCOM_INVITATIONS_DELETE_ENDPOINT,
+    INTERCOM_INVITATIONS_ENDPOINT,
+    INTERCOM_KINDS,
+    INTERCOM_OPEN_ENDPOINT,
+    MIC_VOL_MAX,
+    MIC_VOL_MIN,
+    MSG_VOL_OUTBOUND,
+    OTHER_DOORBELL_VOL_MAX,
+    OTHER_DOORBELL_VOL_MIN,
+    SETTINGS_ENDPOINT,
+    VOICE_VOL_MAX,
+    VOICE_VOL_MIN,
+)
+from ring_doorbell.exceptions import RingError
+from ring_doorbell.generic import RingGeneric
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RingOther(RingGeneric):
+    """Implementation for Ring Intercom."""
+
+    def __init__(self, ring, device_api_id, shared=False):
+        super().__init__(ring, device_api_id)
+        self.shared = shared
+
+    @property
+    def family(self):
+        """Return Ring device family type."""
+        return "other"
+
+    def update_health_data(self):
+        """Update health attrs."""
+        self._health_attrs = (
+            self._ring.query(HEALTH_DOORBELL_ENDPOINT.format(self.device_api_id))
+            .json()
+            .get("device_health", {})
+        )
+
+    @property
+    def model(self):
+        """Return Ring device model name."""
+        if self.kind in INTERCOM_KINDS:
+            return "Intercom"
+        return None
+
+    def has_capability(self, capability):
+        """Return if device has specific capability."""
+        if capability == "open":
+            return self.kind in INTERCOM_KINDS
```

Comment:
Is this the only capability the device has? I think it has motion detection and battery etc no?

Reply:
The Ring Intercom does only 3 things: receives calls, makes voice communication with the connected intercom, and opens the only door connected. 
It has a battery and makes available the battery level, is this a capability to add?

Reply:
Let's leave it as is for now